packages/generators/init-generator.js

```diff
@@ -29,7 +29,14 @@ module.exports = class InitGenerator extends Generator {
 	constructor(args, opts) {
 		super(args, opts);
 		this.isProd = false;
-		this.dependencies = ["webpack", "webpack-cli", "uglifyjs-webpack-plugin"];
+		(this.usingDefaults = false),
+		(this.dependencies = [
+			"webpack",
+			"webpack-cli",
+			"uglifyjs-webpack-plugin",
+			"babel-plugin-syntax-dynamic-import",
+			"path"
```

Comment:
Isn't `path` core Node.js module?

Reply:
Yes, but these are packages that would get installed by npm/yarn, right? Definition part would be in [topScope](https://github.com/webpack/webpack-cli/blob/next/packages/generators/init-generator.js#L69).

Reply:
Yes exactly. `path` needs to be removed